oc/README.md

```diff
@@ -0,0 +1,124 @@
+# Modify Docker config
+
+- Add insecure registries which is required by OpenShift cluster
+
+```bash
+cat << EOF > /etc/docker/daemon.json
+{
+  "insecure-registries" : [ "172.30.0.0/16" ]
+ }
+EOF
+
+systemctl daemon-reload
+systemctl enable docker
+systemctl restart docker
+```
+
+# Install OCP cluster on vm using `oc cluaster up`
+
+- Interesting links :
+  - https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md
+  - https://stefanopicozzi.blog/2016/12/18/getting-started-with-oc-cluster-updown/
+
+# With 3.9, we can't enable service-catlog option
+# -- Registering template service broker with service catalog ... FAIL
+#  Error: cannot register the template service broker
+#  Caused By:
+#    Error: cannot process template openshift-infra/template-service-broker-registration
+#    Caused By:
+#      Error: error processing template "openshift-infra/template-service-broker-registration": namespaces "openshift-template-service-broker" not found
+
+
+```bash
+yum install -y wget
+mkdir -p origin && cd origin
+RELEASE="v3.7.0"
+VERSION="v3.7.0-7ed6862"
+URL="https://github.com/openshift/origin/releases/download/$RELEASE/openshift-origin-server-$VERSION-linux-64bit.tar.gz"
```

Comment:
Is this `URL` variable actually used?

Reply:
It should otherwise this is a mistake. That would be great to get such URL (as I did before at the beginning of the creation of istio ansible playbook) from the github repo release according to the version, os, tar or zip file, ...

As we are mainly interested to use `oc` go client and not `kube-xxx` go clients, we could also just download the the oc go client instead of this 180mb bundle ...